edb/server/main.py

```diff
@@ -239,6 +261,15 @@ async def _run_server(
         ss.init_tls(
             args.tls_cert_file, args.tls_key_file, tls_cert_newly_generated)
 
+        def reload_tls():
+            ss.reload_tls(args.tls_cert_file, args.tls_key_file)
+
+        handler = FileModifiedHandler(reload_tls)
+        observer = watchdog.observers.Observer()
+        observer.schedule(handler, args.tls_cert_file)
+        observer.schedule(handler, args.tls_key_file)
+        observer.start()
```

Comment:
You have to shut it down when we're stopping the server https://python-watchdog.readthedocs.io/en/stable/quickstart.html#a-simple-example